app/models/procedure.rb

```diff
@@ -906,7 +907,8 @@ def ensure_path_exists
 
   def ensure_defaut_groupe_instructeur
     if self.groupe_instructeurs.empty?
-      groupe_instructeurs.create(label: GroupeInstructeur::DEFAUT_LABEL)
+      gi = groupe_instructeurs.create(label: GroupeInstructeur::DEFAUT_LABEL)
```

Comment:
```suggestion
      defaut_groupe_instructeur_create(label: GroupeInstructeur::DEFAUT_LABEL)
```
je pense que tu peux faire ça